packages/storybook/stories/AdhocCalling.stories.mdx

```diff
@@ -0,0 +1,64 @@
+import { MessageBar, MessageBarType } from '@fluentui/react';
+import { Meta, Source } from '@storybook/addon-docs';
+import { DetailedBetaBanner } from './BetaBanners/DetailedBetaBanner';
+
+import AdhocCallingSnippetText from '!!raw-loader!./snippets/AdhocCalling.snippet.tsx';
+
+<Meta
+  id="adhoc calling"
+  title="Concepts/Adhoc calling"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+# Adhoc calling
+
+<DetailedBetaBanner version={'1.6.1-beta.1'} />
+
+## Overview
+
+A call can be started with a Teams user or a call queue by using their id. The id of the Teams user or call queue is
+added to the `CallParticipantsLocator` which is used as the locator in the
+[AzureCommunicationCallAdapterArgs](./?path=/docs/composite-adapters--page#azurecommunicationcalladapterargs)
+when creating your [CallAdapter](path=/docs/composite-adapters--page#call-adapter).
+In the current beta version, only one Teams user can be called. A Teams call queue can also be called. For more
+information on call queues, refer to this
+[Learn documentation](https://learn.microsoft.com/en-us/microsoftteams/create-a-phone-system-call-queue)
+
+<MessageBar messageBarType={MessageBarType.warning}>
+  Note: Teams user ids need to be in format '8:orgid:UUID' when used in the CallParticipantsLocator. For example,
+  '8:orgid:87d349ed-44d7-43e1-9a83-5f2406dee5bd'.
```

Comment:
Is this a real mri? Maybe we can switch this out with 80000000-0000000-00000-00 etc.

Reply:
It is an mri from the Sample tenant.
![image](https://github.com/Azure/communication-ui-library/assets/79475487/568f5723-3c60-4d32-93cc-27c2f5429681)